.azure-pipelines/client.yml

```diff
@@ -50,6 +46,35 @@ jobs:
     displayName: 'Publish Artifacts'
```

Comment:
It won't let leave a comment further up, the Copy task is moving the `dependencies.html` to the artifact staging directory. This no longer happens as the file ain't present anymore. Please adjust the task in the `analyze` job to write directly to the `Build.ArtifactsStagingDirectory`. Delete the `Move Dependencies Report to Build Artifacts` step above.